about.md

```diff
@@ -7,25 +7,32 @@ image: about-background
 
 ## Who are we?
 
-Iron Claw is a [First Robotics Competition](http://www.usfirst.org/roboticsprograms/frc) (FRC)
-team run by the Los Gatos High School Robotics Club. Our team number is 972 and our rookie year is 2016!
-We hope to see all of our fellow teams at Silicon Valley Regional in April!
+Iron Claw is a <a href="http://www.usfirst.org/roboticsprograms/frc">FRC</a>
```

Comment:
This is a markdown page. The proper format for links is `[text](http://example.com)`.